packages/hardhat/contracts/periphery/ControllerHelper.sol

```diff
@@ -530,6 +530,11 @@ contract ControllerHelper is UniswapControllerHelper, AaveControllerHelper, IERC
                 data.collateralToWithdraw.add(data.collateralToFlashloan),
                 data.limitPriceEthPerPowerPerp
             );
+
+            if (address(this).balance > 0) {
+                // convert flashloaned amount + fee from ETH to WETH to prepare for payback
+                IWETH9(ControllerHelperDiamondStorage.getAddressAtSlot(5)).deposit{value: address(this).balance}();
```

Comment:
wonder if we should be consistent here across how we wrap to weth to repay the flashswap

there may be some edge cases we miss if we wrap amount+premium in some cases vs balance in other cases

wrapping the full balance is probably safer, as there could be cases where there is weth that can be used for the premium and eth used for the flashswap